parsons/load.py

```diff
@@ -39,10 +18,16 @@ def load_config(problem_name):
   Returns: The contents of the YAML file as a defaultdict, returning None
       for unspecified attributes.
   """
-  config_files = []
-  for path in PROBLEM_PATHS:
-    config_files.append(os.path.join(os.path.abspath(path), problem_name.lower() + ".yaml"))
-  return load_config_file(config_files)
+  path = os.path.join(os.path.abspath(PARSONS_FOLDER_PATH), problem_name.lower() + ".yaml")
```

Comment:
With the recognition that there is only one path, this logic is shorter, so I combined it with the function above.